### PR TITLE
Add jobs ClientPreServiceTerminate, ClientPreServiceSuspend, ClientPreServiceUnsuspend.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-366.

### CODE REVIEW

File: proc/worker/client.go [hunk 1]
```diff
@@ -290,3 +290,22 @@ func (w *Worker) ClientAfterEndpointMsgSOMCGet(job *data.Job) error {
...
+
+	switch ch.ServiceStatus {
+	case data.ServicePending, data.ServiceActive, data.ServiceSuspended:
+	default:
+		return ErrBadServiceStatus
+	}
+
+	// TODO(maxim) Stop OpenVPN client via dappvpn
+
+	ch.ServiceStatus = data.ServiceTerminated
```
Comment (sofabeat) [inline proc/worker/client.go]: use here https://github.com/Privatix/dappctrl/pull/113/files#diff-83f1cb5a15cd8dac2b3f4a3c7a35d754L124, after PR #113 will be merged

Comment (dzeckelev) [reply proc/worker/client.go]: updated

File: monitor/monitor_test.go [hunk 2]
```diff
@@ -606,52 +606,52 @@ func scheduleTest(t *testing.T, td *testData, queue *mockQueue,
...
-		t.Fatal(err)
-	}
-	el.Data = data.FromBytes(bs)
-	if err := db.Save(el); err != nil {
-		t.Fatal(err)
-	}
-	queue.expect(clientAfterChannelTopUp, func(j *data.Job) bool {
-		return j.Type == data.JobClientAfterChannelTopUp
-	})
+	// TODO: uncomment, when client job handler are completed
```
Comment (ababo) [inline monitor/monitor_test.go]: handlers

File: proc/processor/processor.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package processor
```
Comment (ababo) [inline proc/processor/processor.go]: No need to put the interface in a dedicated package. Just place it inside "proc".

**Pull request closed without merging**